packages/pwa-kit-create-app/assets/bootstrap/js/config/default.js.hbs

```diff
@@ -71,6 +71,10 @@ module.exports = {
             {
                 host: '{{answers.project.commerce.shortCode}}.api.commercecloud.salesforce.com',
                 path: 'api'
+            },
+            {
+                host: '{{answers.project.commerce.instanceUrl}}',
+                path: 'ocapi'
```

Comment:
I think this should be 

```
            }{{~#if answers.project.commerce.instanceUrl}},
            {
                 host: '{{answers.project.commerce.instanceUrl}}',
                 path: 'ocapi'
{{/if}}
            }
```

https://stackoverflow.com/questions/15953293/handlebars-whitespace-control

Reply:
Thanks for the suggestion! I'll wrap the new proxy around that conditional